NetCasbin/NetCasbin.csproj

```diff
@@ -23,8 +18,20 @@
     <FileVersion>1.2.6.0</FileVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PackageId>Casbin.NET</PackageId>
+    <PackageIcon>casbin.png</PackageIcon>
```

Comment:
Where is `casbin.png`?

Reply:
It is here https://github.com/casbin/Casbin.NET/blob/master/NetCasbin/casbin.png.

Reply:
OK